apps/app/src/app/shared/filters/task-status-filter/task-status-filter.component.ts

```diff
@@ -30,6 +29,10 @@ export class TaskStatusFilterComponent
     return this.selectedValue;
   }
 
+  get property() {
```

Comment:
Why is it better written like this?

Reply:
la valeur de property est computed à la demande. Précédemment, la valeur donnée était lors de l'initialisation de la classe donc la value de property était `''` ainsi, le filtre n'était pas envoyé

Reply:
Je n'ai pas compris à quoi sert property ici, c'est utilisé ?

Reply:
Yes, c'est utilisé dans la [génération des filtres](https://github.com/aneoconsulting/ArmoniK.Admin.GUI/blob/main/apps/app/src/app/core/services/pager.service.ts#L82)

Et dans les fitres par defaut de Clarity, c'est property qu'il utilise pour gérer le nom du filtre, donc je me suis aligné !